src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,77 @@
+package shop;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.params.provider.Arguments.*;
+
+class CartTest {
+    RealItem book;
+    VirtualItem flash;
+
+    @BeforeEach
+    void setupData(){
+        book = new RealItem();
+        book.setName("Harry Potter");
+        book.setPrice(150);
+        book.setWeight(200);
+
+        flash = new VirtualItem();
+        flash.setName("Linux");
+        flash.setPrice(15);
+        flash.setSizeOnDisk(25);
+    }
+
+    @Test
+    void realItemAndVirtualItemTotalPriceCheck(){
+       double expectedTotal = 0;
+       double TAX = 0.2;
```

Comment:
please follow java code convention in naming of fields. 
And what the magic number this TAX?